### PR TITLE
Zabbix - Python: library for working with the Zabbix API

### PR DESCRIPTION
**Pull request by matt604-adjust**
For ticket #252

### CODE REVIEW

File: dev-python/pyzabbix/pyzabbix-0.7.5.ebuild [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+RDEPEND="
+	    dev-python/requests[${PYTHON_USEDEP}]
+"
+
+RESTRICT="test"
+
+python_install() {
```
Comment (maxeaubrey) [inline dev-python/pyzabbix/pyzabbix-0.7.5.ebuild]: Not strictly necessary, this is the default :)

File: dev-python/pyzabbix/pyzabbix-0.7.5.ebuild [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+LICENSE="LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+RDEPEND="
+	    dev-python/requests[${PYTHON_USEDEP}]
+"
```
Comment (maxeaubrey) [inline dev-python/pyzabbix/pyzabbix-0.7.5.ebuild]: Nitpicky, but consistent formatting of `DEPEND`/`RDEPEND` is nice and tabs not spaces in ebuilds for whatever reason.

File: dev-python/pyzabbix/pyzabbix-0.7.5.ebuild [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python{2_7,3_6} )
+inherit distutils-r1
+
+DESCRIPTION="Python module for working with Zabbix API"
+HOMEPAGE="https://github.com/lukecyca/pyzabbix"
+SRC_URI="${HOMEPAGE}/archive/${PV}.tar.gz"
```
Comment (maxeaubrey) [inline dev-python/pyzabbix/pyzabbix-0.7.5.ebuild]: Typically (and good practice) should be renamed to full name with ` -> ${P}.tar.gz`.

Comment (matt604-adjust) [reply dev-python/pyzabbix/pyzabbix-0.7.5.ebuild]: Resolved in commit

File: dev-python/pyzabbix/pyzabbix-0.7.5.ebuild (hunk 2 above)
Comment (matt604-adjust) [reply dev-python/pyzabbix/pyzabbix-0.7.5.ebuild]: resolved in commit

**Pull request merged**